web-components/src/components/fixed-footer/BuyFooter.tsx

```diff
@@ -19,73 +19,40 @@ interface BuyFooterProps {
   onClick?: () => void;
 }
 
-const useStyles = makeStyles((theme: Theme) => ({
-  creditsText: {
-    color: theme.palette.info.dark,
-    [theme.breakpoints.up('sm')]: {
-      fontSize: '1.125rem',
-    },
-    [theme.breakpoints.down('sm')]: {
-      fontSize: '0.75rem',
-    },
-  },
-  number: {
-    fontFamily: theme.typography.h1.fontFamily,
-    fontWeight: 800,
-    [theme.breakpoints.up('sm')]: {
-      fontSize: '1.3125rem',
-    },
-    [theme.breakpoints.down('sm')]: {
-      fontSize: '1.125rem',
-    },
-  },
-  creditPrice: {
-    [theme.breakpoints.up('sm')]: {
-      paddingRight: theme.spacing(5),
-    },
-  },
-  buyText: {
-    [theme.breakpoints.up('sm')]: {
-      paddingLeft: theme.spacing(3.5),
-    },
-    [theme.breakpoints.down('sm')]: {
-      paddingLeft: theme.spacing(1.5),
-    },
-  },
-}));
-
 export default function BuyFooter({
   creditPrice,
   href,
   onClick,
 }: BuyFooterProps): JSX.Element {
-  const classes = useStyles({});
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.up('sm'));
 
   return (
     <FixedFooter>
       {creditPrice && (
-        <Grid item className={classes.creditPrice}>
-          <Typography>
-            <span className={classes.number}>${creditPrice.unitPrice}</span>
-            <span className={classes.creditsText}>
-              /credit {creditPrice.currency}
-            </span>
-          </Typography>
+        <Grid item sx={{ pr: { sm: 5 } }}>
+          <Body>
+            <Title as="span" variant="h5">
+              ${creditPrice.unitPrice}
+            </Title>
+            <Body as="span" size="lg" mobileSize="xs" color="info.dark">
```

Comment:
the `as` prop is useful for situations like this, where you want `Body` text but also are using sanity `BlockContent`. Without that, there will likely be console errors for nested `<p>` tags (and those probably exist in this pr - I didn't realize the bug until I was 95% done with the work 😬 )